app/actors/sufia/grant_edit_to_depositor_actor.rb

```diff
@@ -0,0 +1,15 @@
+module Sufia
+  class GrantEditToDepositorActor < CurationConcerns::Actors::GrantEditToDepositorActor
+    private
+
+      # Overriding the parent class so it doesn't grant access
+      # if mediated deposit is enabled
+      def grant_edit_access
+        super unless mediated_deposit?
```

Comment:
Because it's not one of the magick `#update` or `#create` methods that handle chaining? :ok_hand: